### PR TITLE
Docs: Update documentation for search.ignore

### PR DESCRIPTION
**Pull request by benjaoming**
---
:books: Documentation previews :books:

- User's documentation (`docs`): https://docs--10091.org.readthedocs.build/en/10091/

- Developer's documentation (`dev`): https://dev--10091.org.readthedocs.build/en/10091/

### CODE REVIEW

File: docs/user/config-file/v2.rst [hunk 1]
```diff
@@ -751,13 +751,13 @@ check :ref:`config-file/v2:search.ignore`.
 search.ignore
 `````````````
 
-Don't index files matching a pattern.
-This is, you won't see search results from these files.
+List of paths to ignore and exclude from the search index.
+Paths matched will not be seen in search results.
```
Comment (ericholscher) [inline docs/user/config-file/v2.rst:755]:
```suggestion
Paths matched will not be included in search results.
```

**Review by ericholscher — APPROVED** [review]

**Review by humitos — COMMENTED** [review]
I reviewed this from the train, but it seems it didn't go out. I'm sending it now.

File: docs/user/config-file/v2.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (ericholscher) [reply docs/user/config-file/v2.rst]: I would say something like `Paths that match this pattern` or similar to be most clear.

[the remark this answers is not in the document]
Comment (benjaoming) [reply docs/user/config-file/v2.rst]: That definitely reads better!

**Pull request merged**